src/vulkan/rt64_vulkan.cpp

```diff
@@ -53,7 +53,11 @@ namespace RT64 {
 #   elif defined(__ANDROID__)
         VK_KHR_ANDROID_SURFACE_EXTENSION_NAME,
 #   elif defined(__linux__)
-        VK_KHR_XLIB_SURFACE_EXTENSION_NAME,
+#       if defined(RT64_SDL_WINDOW_VULKAN)
+            VK_KHR_WAYLAND_SURFACE_EXTENSION_NAME,
+#       else
+            VK_KHR_XLIB_SURFACE_EXTENSION_NAME,
+#       endif
```

Comment:
This can stay the way it was, just under the `# if !defined(RT64_SDL_WINDOW_VULKAN)`.

Reply:
Ok, it's now as you told me.